decidim-templates/lib/decidim/templates/test/factories.rb

```diff
@@ -5,28 +5,32 @@
 
 FactoryBot.define do
   factory :template, class: "Decidim::Templates::Template" do
+    transient do
+      skip_injection { false }
+    end
     organization
+    name { generate_localized_title(:template_name, skip_injection: skip_injection) }
+    description { generate_localized_title(:template_description, skip_injection: skip_injection) }
     templatable { build(:dummy_resource) }
```

Comment:
Missing (as it is in the original PR)
```suggestion
    templatable { build(:dummy_resource, skip_injection: skip_injection) }
```